sim/shaman/enhancement/rotation.go

```diff
@@ -41,6 +41,15 @@ func (rotation *AdaptiveRotation) DoAction(enh *EnhancementShaman, sim *core.Sim
 	// 	return
 	// }
 
+	if enh.Talents.FeralSpirit {
+		if enh.FeralSpirit.IsReady(sim) {
+			if !enh.FeralSpirit.Cast(sim, target) {
+				enh.WaitForMana(sim, enh.FeralSpirit.CurCast.Cost)
+			}
+			return
+		}
+	}
+
```

Comment:
id have to double check but im not sure if cooldowns go in the rotation or go somewhere else

Reply:
You are correct, and I have fixed this now and removed spirit wolves explicitly from rotation.

It is prioritized above blood lust, and judging timeline, it correctly pops wolves before lust.